StripeConnect/StripeConnect/Source/Internal/Webview/MessageSenders/ReturnedFromSecureWebViewSender.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  ReturnedFromSecureWebViewSender.swift
+//  StripeConnect
+//
+//  Created by Chris Mays on 8/7/24.
+//
+
+import Foundation
+
+struct ReturnedFromSecureWebViewSender: MessageSender {
+    struct Message: Codable, Equatable {
+        let url: String?
```

Comment:
nit: docstrings
```suggestion
        /// The return URL from the `ASWebAuthenticationSession` redirect. This value will be nil if the user canceled out fo the view
        let url: String?
```

Note: I think it's okay that this is a `String` type and not `URL` since its only purpose is to pass onto the JS and never needs to be cast to a URL type